docs/how-monty-works/faq-monty.md

```diff
@@ -81,8 +111,51 @@ While this remains an active area of research in deep-reinforcement learning ([H
 
 Instead, these systems appear to learn complex input-output mappings, which are capable of some degree of composition and interpolation between observed points, but limited generalization beyond the training data. This makes them useful for many tasks, but requires training on enormous amounts of data, and limits their ability to solve benchmarks such as [ARC-AGI](https://github.com/fchollet/ARC-AGI), or more importantly, make themselves very useful when physically embodied. This dependence on input-output mappings means that even approaches such as chain-of-thought or searching over the space of possible outputs (e.g. the [recent o1 models](https://openai.com/index/introducing-openai-o1-preview/)), are more akin to searching over a space of learned "Type 1" actions, rather than the true "Type 2" ([Stanovich and West, 2000](https://www.cambridge.org/core/journals/behavioral-and-brain-sciences/article/abs/individual-differences-in-reasoning-implications-for-the-rationality-debate/2906AEF620B36C10018DD291F790BE97)), model-based thinking that is a marker of intelligence.
+
+## How Does Hierarchical Composition Relate to the Hierarchical Features in Convolutional Neural Networks?
+
+In CNNs, and deep-learning systems more generally, there is often a lack of “object-centric” representations, which is to say that when processing a scene with many objects, the properties of these tend to be mixed up with one another. This is in contrast to humans, where we understand the world as being composed of discrete objects with a degree of permanence, and where these objects have the ability to interact with one another.
+
+Furthermore, any given object in our brain is represented spatially, where the shape of the object - i.e. the relative arrangement of features - is far more important than low-level details like a texture that might correlate with a particular object class. Again, this is different from how CNNs and other deep-learning systems learn to represent objects.
```

Comment:
Could give the example of features arranged in the shape of a face here to make it a bit more concrete

Reply:
Good idea, I added the actual painting.